qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -141,3 +142,12 @@ def _set_interleaved_element(self, interleaved_element):
                     interleaved_element.name
                 )
             ) from error
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        options = super()._default_experiment_options()
+        # Computation of EPG is not necessary for IRB.
+        # This will drastically reduce overhead of ops counting.
+        options.gate_error_ratio = "skip"
```

Comment:
It is tough due to backward compatibility because the experiment data loaded from experiment db doesn't have this value so we cannot distinguish intentional `None` and "not exist" (in this case we need to populate the value for backward compatibility).